OWNERS_ALIASES

```diff
@@ -125,6 +125,8 @@ aliases:
     - claudiubelu
     - marosset
     - zylxjtu
+  wg-api-expression-leads:
+    - aaron-prindle
```

Comment:
will need another lead for 2+

Reply:
Great point, +1 on getting an additional lead here.  I am currently in the process of finding an addition SIG lead. Will update the thread here and PR when this is done.